### PR TITLE
feat: add appearance property for material theme

### PR DESCRIPTION
**Pull request by kTkaciovaite**
Discussion: https://github.com/ng-select/ng-select/issues/1334

### CODE REVIEW

**Comment by varnastadeus** [comment]
Thx for contribution! Can you add an image to visual changes?

**Comment by kTkaciovaite** [comment]
Sure, so nothing has changed for Default and Ant Design theme. For material theme, a new config property (appearance) was introduced. There are two options for appearance property:

1. underline (default). Styling did not change for this option.
Initial:
image

Focused:
image

Selected:
image

2. outline.
Initial:
image

Focused:
image

Selected:
image

File: src/ng-select/lib/ng-select.component.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
     @Input() addTagText: string;
     @Input() loadingText: string;
     @Input() clearAllText: string;
+    @Input() appearance: string;
```
Comment (varnastadeus) [inline src/ng-select/lib/ng-select.component.ts]: add this new input to the readme, and point that it only controls material theme appearance at the time

Comment (kTkaciovaite) [reply src/ng-select/lib/ng-select.component.ts]: README.md updated

**Pull request merged**